### ISSUE TITLE
rake db:migrate does not work

### ISSUE THREAD

**Issue by cdelavar** [learn-co-curriculum/Active-Record-Association-Methods#6]
The lab says that we should be able to run rake db:migrate the first thing in the lab, but when I run it, none of the tests pass and it throws a lot of errors. Can you please fix this?

**Comment by aturkewi** [comment]
The Gemfile does not call for `gem 'rake'`. Check with @jinjo39 , and this did NOT solve the problem, but I believe this should still be added.

**Comment by bhollan** [comment]
I was having an issue that may or may not be the same.  It gave me 100+ warnings/errors about an already initialized constant Rake::VERSION  Rake::Version::MAJOR, Rake::Version::MINOR,...etc

As per [this SO thread](http://stackoverflow.com/questions/7319262/stack-level-too-deep-running-rake-dbcreateall/7319389#7319389) I checked and I had a 3 versions of a 'rake' gem installed.  I did `gem uninstall rake`, then when I ran `rake db:migrate` I got a much more helpful and sensible error.

Also I see the "include solution" commit message on the stray/extra `migrate` folder under the repo's root directory.  Not sure if that's intentional or not.
